CMakeLists.txt

```diff
@@ -249,6 +249,9 @@ if(NOT blue_framework_POPULATED)
 	set_target_properties(UpdateLibsAndDLLsForCurrentBuildConfiguration PROPERTIES FOLDER "BlueFramework/Commands")
 endif()
 
+set (CMAKE_PREFIX_PATH "C:/OSGeo4W64/share/cmake/proj4")
+find_package(PROJ4 REQUIRED)

```

Comment:
PROJ4 was superseeded by PROJ. Please update to the newest version as described [here](https://proj.org/development/cmake.html).

Reply:
Resolved: efdc4b3